### PR TITLE
Move eligible_voters to user_roles_controller

### PR DESCRIPTION
**Pull request by danieljames-dj**

### CODE REVIEW

**Comment by danieljames-dj** [comment]
Ready for review, but https://github.com/thewca/worldcubeassociation.org/pull/9121 needs to be merged before this so marking this PR as draft.

**Comment by danieljames-dj** [comment]
The above mentioned PR is merged now, so ready for review.

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -382,7 +382,7 @@ def do_override_regional_records
   end
 
   def all_voters
-    voters User.eligible_voters, "all-wca-voters"
+    voters Api::V0::UserRolesController.eligible_voters, "all-wca-voters"
```
Comment (gregorbg) [inline app/controllers/admin_controller.rb:385]: I am not happy about calling controllers from other controllers.
Is there a reasonable place in `app/models` or in `lib` where you can store this code?

**Review by gregorbg — CHANGES_REQUESTED** [review]

File: app/controllers/admin_controller.rb (hunk 1 above)
Comment (danieljames-dj) [reply app/controllers/admin_controller.rb]: Maybe I'll have it in `user` model itself for now. I was thinking to make it as an API for the react migration, but will do that later.

**Review by gregorbg — APPROVED** [review]

**Pull request merged**